src/Microsoft.Framework.Runtime.Roslyn.Interfaces/project.json

```diff
@@ -15,5 +15,13 @@
                 "System.Runtime": "4.0.20-beta-*"
             }
         }
+    },
+
+    "scripts": {
+        "postbuild": [
+            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/kre-clr-win-x86/bin",
```

Comment:
I wonder if we should just have a "build/placebinaries" script that does all three of these commands for you. Also, it would be nice if we could also put the binaries into the right place for x64 builds. That's a side-issue though.